Basket-Donation-Payments/Basket-Donation-Payments/View Controllers/NonprofitSignupViewController.m

```diff
@@ -18,28 +19,58 @@ @interface NonprofitSignupViewController ()
 @property (weak, nonatomic) IBOutlet UITextField *nonprofitCategoryTextField;
 @property (weak, nonatomic) IBOutlet UITextField *nonprofitWebsiteTextField;
 //TODO: set up with Paypal, Braintree?
-
+@property (nonatomic) BOOL finishedSavingBoth;
 
 @end
 
 @implementation NonprofitSignupViewController
 
 - (void)viewDidLoad {
     [super viewDidLoad];
+    self.finishedSavingBoth = false;
```

Comment:
In Objective-C the `BOOL` type values are defined as `YES` and `NO` instead of the traditional `true` and `false` from other programming languages. Although you can still use `true` and `false` or `1` and `0`, it's a good practice to use `YES` and `NO` as defined in the `BOOL` type.

Reply:
Got it, thanks! I forgot to do that here